package.json

```diff
@@ -25,7 +25,7 @@
 		"eslint:fix": "eslint --ext .js,.jsx,.ts,.tsx . --fix",
 		"stylelint": "stylelint \"app/**/*.css\" \"client/**/*.css\" \"app/**/*.less\" \"client/**/*.less\" \"ee/**/*.less\"",
 		"stylelint:fix": "stylelint --fix \"app/**/*.css\" \"client/**/*.css\" \"app/**/*.less\" \"client/**/*.less\" \"ee/**/*.less\"",
-		"typecheck": "NODE_OPTIONS=\"--max-old-space-size=4092\" tsc --noEmit --skipLibCheck",
+		"typecheck": "NODE_OPTIONS=\"--max-old-space-size=8184\" tsc --noEmit --skipLibCheck",
```

Comment:
4G was  already high IMO.. is this really needed to be 8G ? not everybody 8G RAM for NodeJS 😬

Reply:
I was getting "Heap out of memory" when running the type check, since I doubled the memory. I think as we're increasing the typescript definitions throughout the codebase (like in this PR), more memory will be needed to run these tests. 

But let me know if you find some other solution to solve the out of memory issue